pipelines/vars/foreman_nightly.yml

```diff
@@ -0,0 +1,3 @@
+forklift_name: "pipeline-foreman-nightly-{{ foreman_os }}"
```

Comment:
Should this be `pipeline-foreman-{{ foreman_repositories_version }}-{{ foreman_os }}`